frontend/src/components/input/TextEditor.vue

```diff
@@ -141,7 +142,6 @@ const editor = new Editor({
     emit("updateValue", editor.getHTML())
   },
   onFocus: () => (hasFocus.value = true),
-  onBlur: () => (hasFocus.value = false),
```

Comment:
This was moved to line 454 `focusout` as we don't want to hide the buttons if the focus moves from the editor input to the menu bar.